cypress/integration/playground/application-shell.js

```diff
@@ -6,8 +6,10 @@ import { URL_BASE, URL_STATE_MACHINES } from '../../support/urls';
 describe('when user is authenticated', () => {
   it('should log out with reason "user"', () => {
     cy.login({ redirectToUri: URL_STATE_MACHINES });
-    cy.findByLabelText('open menu').click();
+
+    cy.findByRole('button', { name: /close user settings menu/i }).click();
```

Comment:
We call this user settings menu to my knowledge. Now w ecan target a button and the label instead of label text (which is strange in of itself).